include/dlr_pipeline.h

```diff
@@ -0,0 +1,64 @@
+#ifndef DLR_PIPELINE_H_
+#define DLR_PIPELINE_H_
+
+#include <graph/graph_runtime.h>
+#include <tvm/runtime/memory.h>
+#include "dlr_common.h"
+
+#if defined(_MSC_VER) || defined(_WIN32)
+#define DLR_DLL __declspec(dllexport)
+#else
+#define DLR_DLL
+#endif  // defined(_MSC_VER) || defined(_WIN32)
+
+namespace dlr {
+
+/*! \brief class PipelineModel
+ */
+class DLR_DLL PipelineModel : public DLRModel {
+ private:
+  int count_;
+  std::vector<DLRModelPtr> dlr_models_;
+  void SetupPipelineModel(std::vector<DLRModelPtr>& dlr_models);
+
+ public:
+  /*! \brief Load model files from given folder path.
+   */
+  explicit PipelineModel(std::vector<DLRModelPtr>& dlr_models, const DLContext& ctx)
```

Comment:
Let's make `std::vector<DLRModelPtr>& dlr_models` -> `const std::vector<DLRModelPtr>& dlr_models`

Can we initialize `dlr_models_` here instead since we are just copying `dlr_models` to `dlr_models_`:
```
  : DLRModel(ctx, DLRBackend::kPIPELINE), dlr_models_(dlr_models) {
```

That way we don't need to pass to SetupPipelineModel

Reply:
fixed